blueocean-rest-impl/pom.xml

```diff
@@ -124,8 +124,8 @@
             <scope>test</scope>
         </dependency>
         <dependency>
-            <groupId>com.github.tomakehurst</groupId>
-            <artifactId>wiremock-jre8-standalone</artifactId>
+            <groupId>org.wiremock</groupId>
```

Comment:
Not strictly necessary for this PR, but getting rid of remaining Java 8 references as we move to Java 17 felt wise.